spec/models/dinner_table_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe DinnerTable, type: :model do
+  let!(:dinner_table) {FactoryBot.create :dinner_table}
+
+  describe "Associations" do
+    it {expect(dinner_table).to have_many(:orders).dependent(:destroy)}
+  end
+
+  describe "Enums" do
+    it {expect(dinner_table).to define_enum_for(:status).with_values([:free, :using])}
+  end
+
+  describe ".filter_status" do
+    it "matching free" do
+        expect(DinnerTable.filter_status(0).first.status).to eq("free")
```

Comment:
dùng subject ở bên ngoài
subject{DinnerTable.filter_status(0).first.status}
bên trong it thì chỉ cần is_expected thôi e
tránh lặp code

Reply:
Dạ vâng